src/cli/subcommands/attach_cmd.rs

```diff
@@ -326,11 +327,6 @@ impl AttachCommand {
         bind_request!(context, api,
                 // Node API
                 "node_status" => |()| ApiInfo::node_status_req(),
-
-                // Sync API
-                "sync_check_bad" => ApiInfo::sync_check_bad_req,
```

Comment:
Should we put `forest-cli attach` behind a feature if we disable most of the APIs? `boa-engine` takes quite long to compile  cc @elmattic

Reply:
Defining a pattern for adding these back is tracked in #4032, so there's a chance it would be feature gated only to be re-enabled. I don't mind - your argument is compelling